runtimes/python/python_apps.md

```diff
@@ -78,36 +74,19 @@ of your project with a Flask `app` object inside.
 
 Basically, you should just point to a WSGI capable object.
 
-
 ### Celery apps
 
-We also support celery apps out of the box. To deploy a celery app, use the key `celery` in `deploy` like the following:
+We also support celery apps out of the box. To deploy a celery app, use the
+`CC_PYTHON_CELERY_MODULE` variable:
 
-```json
-   {
-      "deploy": {
-         "celery": {
-            "module": "mymodule"
-         }
-      }
-   }
 ```
-
-You can activate `beat` or add some more configurations like this
-
-```json
-   {
-      "deploy": {
-         "celery": {
-            "module": "mymodule",
-            "log": "/path/to/logdir",
-            "beat": "yes"
-         }
-      }
-   }
+CC_PYTHON_CELERY_MODULE="mymodule"
 ```
 
-The `log` path will be concatenated to the application's path.
+You can also activate beat with `CC_PYTHON_CELERY_USE_BEAT=true` and provide a given log
+dir for celery with `CC_PYTHON_CELERY_LOGFILE="/path/to/logdir"`.
+
+The `CC_PYTHON_CELERY_LOGFILE` path will be concatenated to the application's path.
```

Comment:
I'd rather have "the `CC_PYTHON_CELERY_LOGFILE` path is relative to the application path". It's more user centric.